packages/wallet-sdk/package.json

```diff
@@ -53,6 +53,7 @@
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-unused-imports": "^3.0.0",
+    "glob": "^11.0.0",
```

Comment:
adding it as dev dep. yarn 3.6 complains this is not found otherwise

Reply:
Maybe in a future PR we can work on upgrading to yarn 4+. Not critical for now